README.md

```diff
@@ -615,6 +615,7 @@ Here are some companies/projects (alphabetical order) using OpenAPI Generator in
 - [Myworkout](https://myworkout.com)
 - [NamSor](https://www.namsor.com/)
 - [Openet](https://www.openet.com/)
+- [openVALIDATION] (https://openvalidation.io/)
```

Comment:
Please remove the space. 😃 

```suggestion
- [openVALIDATION](https://openvalidation.io/)
```

Preview: 

![image](https://user-images.githubusercontent.com/1885716/76614076-b1a80380-6562-11ea-8d22-b33e3c9baa1b.png)